libs/common-const/src/common.ts

```diff
@@ -68,22 +68,19 @@ export const ORDER_BOOK_API_UPDATE_INTERVAL = ms`30s` // 30s
 export const MINIMUM_ORDER_VALID_TO_TIME_SECONDS = 120
 // Minimum deadline for EthFlow orders. Like the default deadline, anything smaller will be replaced by this
 export const MINIMUM_ETH_FLOW_DEADLINE_SECONDS = 600 // 10 minutes in SECONDS
-export const MINIMUM_ETH_FLOW_SLIPPAGE_BPS: Record<SupportedChainId, number> = {
+
+export const DEFAULT_ETH_FLOW_SLIPPAGE_BPS: Record<SupportedChainId, number> = {
   [SupportedChainId.MAINNET]: 200, // 2%
   [SupportedChainId.GNOSIS_CHAIN]: DEFAULT_SLIPPAGE_BPS,
   [SupportedChainId.ARBITRUM_ONE]: DEFAULT_SLIPPAGE_BPS,
   [SupportedChainId.SEPOLIA]: DEFAULT_SLIPPAGE_BPS,
 }
+
 export const MINIMUM_ETH_FLOW_SLIPPAGE: Record<SupportedChainId, Percent> = mapSupportedNetworks(
-  (chainId) => new Percent(MINIMUM_ETH_FLOW_SLIPPAGE_BPS[chainId], 10_000)
+  (chainId) => new Percent(DEFAULT_ETH_FLOW_SLIPPAGE_BPS[chainId], 10_000)
```

Comment:
Sorry, I meant both of these env vars should remain `minimum`, as one is derived from the other.